harness/deepEqual.js

```diff
@@ -6,268 +6,322 @@ description: >
 defines: [assert.deepEqual]
 ---*/
 
-var deepEqual = (function () {
```

Comment:
this is now `assert.deepEqual._compare` to avoid exposing extra names to the global